wall_tracking_node:
  ros__parameters:
    topic_measurements_in: "wall_poses"
    topic_landmarks_out: "target_tracking/landmarks"
    clutter_rate: 0.001
    probability_of_detection: 0.7
    probability_of_survival: 0.99
    gate_threshold: 2.5
    min_gate_threshold: 1.0
    max_gate_threshold: 10.0
    confirmation_threshold: 0.1
    deletion_threshold: 0.2
    initial_existence_probability: 0.4
    std_position: 0.2
    std_sensor: 0.5
    update_interval_ms: 100
    fixed_frame: "map"